### PR TITLE
Panic bunker + new player notification configs

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Panic bunker! Uses the power of the database to reject new ckeys that have never connected to the server(s) before.
Admins are exempt, even while deadmin'ed, but they are also unlikely to be new ckeys except in rare cases that involve vv fuckary.

Its both a round only +server verb, and a config option for more long term panic attacks.

New player notification is now configurable. You can change how long they happen for.

Ie: First connection only, first x number of days on the server or never

Adds an optional default disabled irc notification of a players first time connecting.
This will only fire if first connection admin alerts above are turned on AND there are no valid admins on and not afk.

### CODE REVIEW

File: code/modules/admin/verbs/panicbunker.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/client/proc/panicbunker()
+	set category = "Server"
+	set name = "Toggle Panic Bunker"
+	if (!config.sql_enabled)
+		usr << "<span class='adminnotice'>The Database is not enabled!</span>"
+		return
+
+	if (!dbcon || !dbcon.IsConnected())
```
Comment (MrStonedOne) [inline code/modules/admin/verbs/panicbunker.dm]: todo: make it so you can turn it off while the database is not connected, or flat out remove some of these checks

**Comment by paprka** [comment]
it should redirect them to a hub server too come on you need to go full /vg/

**Comment by MrStonedOne** [comment]
@paprka if you are just gonna shit post on my prs just fucking go.

What is the point? stalk me around github? Is that suppose to annoy me?

Because its not going to work.

All i'm going to do is go to github and report you for harassing me.

You've already been kicked out from this community, the only reason you can still comment here is because github a shit and requires you to put up a support ticket to issue bans to repos attached to organisations rather than personal accounts.

**Comment by Dorsisdwarf** [comment]
>paprka hounded out of the community by the lynch mob

>Still makes useful ease-of-life PRs for the community

>"Halp he is stalk me"

**Comment by dannno** [comment]
is this really what you think harassment is
where do you think we are

**Comment by MrStonedOne** [comment]
Alright, I did a bunch more testing while on more sleep, should be ready once this hits 24 hours.

**Comment by Razharas** [comment]
What for such a thing needed?

**Comment by MrStonedOne** [comment]
Admins have requested it. Every now and then we get a really determined ban evader, who figured out how to spoof cids or some shit. so they just keep connecting and shit.

this would allow admins to temporarily disable new users from connecting to the server while not preventing their regulars from connecting.

Also helps for when the game gets links on major sites to prevent the wave of griffers

**Pull request merged**